src/helpers/system.js

```diff
@@ -66,6 +92,9 @@ module.exports = {
 
   getShellInfo: () => {
     utils.log('trace', 'getShellInfo', process.env);
+    if (isAlpine()) {
```

Comment:
Why is alpine special here? I think it's because it's ash/busybox based, however alpine is not the only distro to use this or similar combos. I think maybe adding a broader detection for ash/busybox might be better, and the alpine specific case will be covered, in addition to other light distros.